### PR TITLE
Add client aws_opensearch

### PR DESCRIPTION
**Pull request by cydrain**

### CODE REVIEW

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]
Please add opensearch related info in `README.md` and the python package in `pyproject.toml`

File: vectordb_bench/backend/clients/aws_opensearch/config.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+from ..api import DBConfig, DBCaseConfig, MetricType, IndexType
+
+
+class AWSOpenSearchConfig(DBConfig, BaseModel):
+    host: str = (
+        "xxxxxx.us-west-2.es.amazonaws.com"
+    )
+    port: int = 443
+    user: str = "admin"
+    password: SecretStr = "xxxxxx"
```
Comment (XuanYang-cn) [inline vectordb_bench/backend/clients/aws_opensearch/config.py:13]: these default values will be display and used in front page, please just leave it empty
image

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]

File: vectordb_bench/backend/clients/aws_opensearch/config.py (hunk 1 above)
Comment (cydrain) [reply vectordb_bench/backend/clients/aws_opensearch/config.py]: done

**Review by XuanYang-cn — APPROVED** [review]

**Comment by navneet1v** [comment]
@XuanYang-cn this PR lacks optimal settings for AWS Opensearch to give best performance. What is the best way to add those improvements here?

**Comment by xiaofan-luan** [comment]
@navneet1v 
feel free to optimize and submit another request.
We are not expert on optimizing opensearch.

Or you can give some critical optimization?
I'm not sure Disable the refresh interval and disable replica will a an option in production.

**Comment by liliu-z** [comment]
@alwayslove2013 let's take a closer look at this

**Pull request merged**